db/migrate/20241023054414_create_users.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUsers < ActiveRecord::Migration[7.2]
+  def up
+    create_table :users do |t|
+      t.string "name"
+      t.string "email"
+      t.integer "role", default: 2
+
+      t.string "password_digest"
+      t.timestamps
```

Comment:
don't we need to do indexing for any column?

Reply:
Bhai, the email column  is being used as an index. I also added a migration for adding the role as an index. There was slight error in adding the index in the user migration, i fixed that.
Thank you.